### PR TITLE
[Native] Add approx_distinct e2e tests for decimal type

### PR DESCRIPTION
**Pull request by karteekmurthys**
## Description
Add approx_distinct e2e tests for decimals.

## Motivation and Context
Required for running `analyze` command on tables with decimal columns.

## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== RELEASE NOTES ==

General Changes
* ...
* ...

Hive Changes
* ...
* ...
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by aditi-pandit — COMMENTED** [review]
Thanks @karteekmurthys. Took a quick look.

File: presto-main/etc/catalog/hana.properties
[the remark this answers is not in the document]
Comment (karteekmurthys) [reply presto-main/etc/catalog/hana.properties:1]: Undone.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java [hunk 1]
```diff
@@ -248,6 +248,20 @@ public void testAnalyzeStats()
                             "('regionkey', NULL, 1.0, 0.0, NULL, '4', '4')," +
                             "('nationkey', NULL, 1.0, 0.0, NULL, '11', '11')," +
                             "(NULL, NULL, NULL, NULL, 1.0, NULL, NULL))");
+
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:251]: Split this and above logic into individual methods.

**Comment by aditi-pandit** [comment]
@karteekmurthys : Can you remove the Velox submodule changes from this PR ? They are not needed anymore.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java [hunk 2]
```diff
@@ -1206,6 +1226,45 @@ public void testDecimalRangeFilters()
         }
     }
 
+    @Test
+    public void testDecimalApproximateAggregates()
+    {
+        // Short decimals with nulls.
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:1232]: Nit : Remove this line. The test has both Short and Long decimals.

**Review by aditi-pandit — COMMENTED** [review]
Thanks @karteekmurthys. Minor nit.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java (hunk 2 above)
Comment (karteekmurthys) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java]: Done.

**Review by aditi-pandit — APPROVED** [review]
Thanks @karteekmurthys

**Pull request merged**